packages/website/src/snippets/Layout/ColumnsContentSizedColumn.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+import { Column, Columns, Placeholder } from "..";
+
+export default function ColumnsContentSizedColumn() {
+  return (
+    <Columns space={24}>
+      <Column width="content">
+        <Placeholder height={200} label="I'm a content-sized column" />
+      </Column>
+      <Column>
```

Comment:
useless? Or we want it here for documentation on using a Column without props?

Reply:
yep, probably those examples were written before the "auto-insertion" of `Column`. Fixing them